### PR TITLE
implement This Page Intentionally Left Blank

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

**Comment by mikeconley** [comment]
I dig this feature.

File: lib/blank.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+
+  if (tab.url !== 'about:blank') {
+    return;
+  }
+
+  // worker = tab.attach({
+  //   contentScriptFile: self.data.url('newtab-content.js'),
+  //   contentScriptOptions: { 'bucket': telemetry.bucket, 'icon': self.data.url('chatheads.svg') }
+  // });
```
Comment (bwinton) [inline lib/blank.js]: We should probably remove these comments, but I'm happy to do it in a follow-up patch.  :)

File: data/blank.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+body::after {
+  content: "THIS PAGE INTENTIONALLY LEFT BLANK.";
+  font-family: 'Lucida Grande', Arial, sans-serif;
+  position: absolute;
+  bottom: 1em;
+  left: 50%;
+  margin-left: -9.5em;
```
Comment (bwinton) [inline data/blank.css]: Nice solution to getting it centered!  :smile:

**Comment by bwinton** [comment]
I'll merge this tomorrow, and delete the comments if you haven't gotten to them by then.

Thanks!

**Pull request merged**